packages/codelabs/src/intermediate/lit-html.md

```diff
@@ -1070,36 +1221,49 @@ html`
 <!DOCTYPE html>
 <html>
   <body>
-    <news-app></news-app>
+    <hn-comment-app></hn-comment-app>
 
     <script type="module">
       import { LitElement, html } from 'https://unpkg.com/lit-element?module';
       import 'https://unpkg.com/@material/mwc-button?module';
 
-      class NewsApp extends LitElement {
+      class HackerNewsCommentApp extends LitElement {
         static get properties() {
           return {
             loading: { type: Boolean },
-            articles: { type: Array },
+            comments: { type: Array },
             filter: { type: String },
           };
         }
 
         connectedCallback() {
           super.connectedCallback();
 
-          if (!this.articles) {
-            this.fetchArticles();
+          if (!this.comments) {
+            this.fetchComments();
           }
         }
 
-        async fetchArticles() {
+        async fetchComments() {
           this.loading = true;
-          const response = await fetch(
-            'https://newsapi.org/v2/everything?q={some-topic}&apiKey={your-api-key}',
-          );
-          const jsonResponse = await response.json();
-          this.articles = jsonResponse.articles;
+          const path = "https://hacker-news.firebaseio.com/v0/item";
+          const itemId = 22115410;
+
+          let idx = 0;
+          const maxItems = 5;
+          const array = [];
+
+          while (array.length < maxItems) {
+            await fetch(`${path}/${itemId - idx}.json`)
```

Comment:
I'd prefer to stay consistent and either use async await everywhere, or use .thens, so we should probably go with async await

Reply:
consistency is better i agree. I will update it to the async/await schema